src/assets/sections.json

```diff
@@ -0,0 +1,28 @@
+[
+  "arts",
+  "automobiles",
+  "books",
+  "business",
+  "fashion",
+  "food",
+  "health",
+  "home",
+  "insider",
+  "magazine",
+  "movies",
+  "nyregion",
+  "obituaries",
+  "opinion",
+  "politics",
+  "realestate",
+  "science",
+  "sports",
+  "sundayreview",
+  "technology",
+  "theater",
+  "t-magazine",
+  "travel",
+  "upshot",
+  "us",
+  "world"
+]
```

Comment:
It is possible that I'll be able to replace this with something dynamic, but I've not discovered an easy way to do that quite yet.